crates/subspace-node/src/domain/evm_chain_spec.rs

```diff
@@ -147,82 +147,86 @@ pub fn devnet_config<F: Fn() -> GenesisConfig + 'static + Send + Sync>(
+        SpecId::Gemini => {
+            let sudo_account = AccountId::from_str("f31e60022e290708c17d6997c34de6a30d09438f")
+                .expect("Invalid Sudo account");
+            testnet_genesis(
+                vec![
+                    // Genesis operator
+                    AccountId::from_str("2ac6c70c106138c8cd80da6b6a0e886b7eeee249")
+                        .expect("Wrong executor account address"),
+                    // Sudo account
+                    sudo_account,
+                ],
+                Some(sudo_account),
+                Default::default(),
+                1002,
+            )
         }
-        "devnet" => {
-            let constructor = move || {
-                let sudo_account = AccountId::from_str("b66a91845249464309fad766fd0ece8144547736")
-                    .expect("Invalid Sudo account");
-                testnet_genesis(
-                    vec![
-                        // Genesis operator
-                        AccountId::from_str("cfdf9f58d9e532c3807ce62a5489cb19cfa6942d")
-                            .expect("Wrong executor account address"),
-                        // Sudo account
-                        sudo_account,
-                    ],
-                    Some(sudo_account),
-                    vec![(
-                        sudo_account,
-                        AccountId::from_str("5b267fd1ba3ace6e3c3234f9576c49c877b5beb9")
-                            .expect("Wrong relayer account address"),
-                    )],
-                    1003,
-                )
-            };
-            devnet_config(constructor)
+        SpecId::DevNet => {
+            let sudo_account = AccountId::from_str("b66a91845249464309fad766fd0ece8144547736")
+                .expect("Invalid Sudo account");
+            testnet_genesis(
+                vec![
+                    // Genesis operator
+                    AccountId::from_str("cfdf9f58d9e532c3807ce62a5489cb19cfa6942d")
+                        .expect("Wrong executor account address"),
+                    // Sudo account
+                    sudo_account,
+                ],
+                Some(sudo_account),
+                vec![(
+                    sudo_account,
+                    AccountId::from_str("5b267fd1ba3ace6e3c3234f9576c49c877b5beb9")
+                        .expect("Wrong relayer account address"),
+                )],
+                1003,
+            )
         }
-        "" | "local" => {
-            let constructor = move || {
-                testnet_genesis(
-                    accounts.clone(),
-                    // Alith is sudo
-                    Some(accounts[0]),
-                    vec![(accounts[0], accounts[0]), (accounts[1], accounts[1])],
-                    1001,
-                )
-            };
-            local_testnet_config(constructor)
+        SpecId::Local => {
+            testnet_genesis(
+                accounts.clone(),
+                // Alith is sudo
```

Comment:
```suggestion
                // Alice is sudo
```

Reply:
This was introduced by me. Alith here is generally used for EVM based chains on polkadot js

Reply:
Interesting, good to know.